addons/mail/static/src/messaging/entity/activity/activity.js

```diff
@@ -42,6 +42,16 @@ function ActivityFactory({ Entity }) {
             }
         }
 
+        /**
+         * @returns {boolean}
+         */
+        get isCurrentPartnerAssignee() {
+            if (!this.assignee) {
+                return false;
+            }
+            return this.assignee.partner === this.env.messaging.currentPartner;
+        }
```

Comment:
It's funny, I just introduced `currentUser` in my field branch, we're gonna have some fun with using sometimes `Partner` and sometimes `User`.

You can keep it like this for now, but just bear in mind that we may not be aware of the partner of a `User` in JS, so `this.assignee.partner` could be `undefined`. Also, if we remove `currentPartner` from messaging, then this condition is truthy, which is weird... ^^'

Reply:
There is a test using this condition when false and when true, so if it's always truthy it'll be detected :)

Reply:
I mean, always truthy when at least `assignee` exist but we are not aware of partner yet and there is no current partner right now, because `undefined === undefined`